service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java

```diff
@@ -38,7 +39,10 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             data.put("hearingDetails", removeHearingLinkedToCmo(caseData.getHearingDetails(),
                 caseData.getSealedCMOs().get(1).getId()));
 
+
             data.put("sealedCMOs", removeCaseManagementOrder(caseData.getSealedCMOs()));
```

Comment:
Make sure to remove the changes to CMOs

Reply:
and the linked hearing above

Reply:
This is fine, DM was never actually performed on SA20C50002 yet. The DM is remove a CMO and to remove a final order.